src/types.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  B3Propagator,
+  HttpCorrelationContext,
+  HttpTraceContext,
+} from '@opentelemetry/core';
+import { NodeSDKConfiguration } from '@opentelemetry/sdk-node';
+
+/** Lightstep configuration (build on NodeSDKConfiguration) */
+export interface LightstepNodeSDKConfiguration extends NodeSDKConfiguration {
+  token?: string;
+  spanEndpoint?: string;
+  serviceName: string;
+  serviceVersion?: string;
+  propagators?: string;
```

Comment:
It's a comma separated string of propagator names and users can specify more than one (built-in) propagator. The main use case is so that a user can specify OTEL_PROPAGATORS=b3,tracecontext and have a composite propagator with both setup. Since LightstepNodeSDKConfig extends NodeSDKConfiguration, and it has a `httpTextPropagator` option, that is the option users should use if they want to configure a propagator using an object instance of some kind. I can make sure to short circuit our setup if it's set.